RoboFile.php

```diff
@@ -66,6 +77,24 @@ private function processPackagePipeline(string $package): ?Result
             $task->exec('composer config prefer-stable false');
         }
 
+        $composerData = json_decode(file_get_contents($composerJsonPath), true);
```

Comment:
The question is how fail-proof this code should be?

What if `$composerJsonPath` exists but unreadable, which will return `false` and `json_decode` will crash on `false` call. (in strict mode).